server/controller/user/updateUserController.js

```diff
@@ -0,0 +1,35 @@
+const { updateUserQuery } = require('../../database/queries');
+const { boomify, updateUserValidation } = require('../../utilis');
+
+const updateUserController = async (req, res, next) => {
+  try {
+    const { userId } = req;
+    const {
+      firstName, lastName, mobile, address,
+    } = await updateUserValidation.validateAsync(req.body, {
+      abortEarly: false,
+    });
+
+    const { rows } = await updateUserQuery({
+      firstName, lastName, mobile, address, userId,
+    });
+
+    if (!rows.length) {
+      throw boomify(400, 'update failed');
+    }
+
+    res.status(200).json({
+      statusCode: 200,
+      message: 'user updated Successfully',
+      data: rows,
```

Comment:
please see this [link](https://github.com/GSG-G9/beauty-salon/pull/66#discussion_r605964833)

Reply:
I changed the query so that it returns specific data and in the controller, I only sent rows[0] because to make the controller file more readable.